nncf/torch/quantization/algo.py

```diff
@@ -728,20 +763,21 @@ def _build_insertion_commands_list_for_quantizer_setup(self,
 
         for qp_id in non_unified_scales_quantization_point_ids:
             qp = quantizer_setup.quantization_points[qp_id]
-            ip = qp.insertion_point
+            qip = qp.insertion_point
+            tp = PTTargetPointTranslator.translate(qip)
             qconfig = quantizer_setup.quantization_points[qp_id].qconfig
```

Comment:
@alexsu52 FYI, this is how the quantizer insertion points generated by the common algo will be translated into `PTTargetPoint` in PT. I suggest you take a similar approach in TF.

Reply:
@vshampor It looks good for me. @andrey-churkin What do you think?